lib/logstash/outputs/bson/big_decimal.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2009-2014 MongoDB Inc.
```

Comment:
Well most of the code is pasted from ruby-bson.  I don't know the rules very well. So I put it in so we can discuss whether it should stay.  If it was not there no one would question the missing copyright text :-)